webpack/envs/ciConfig.js

```diff
@@ -37,9 +36,5 @@ const plugins = {
 export const ciConfig = {
   mode: NODE_ENV,
   devtool: false,
-  plugins: [
-    plugins.duplicatesReport,
-    process.env.BUNDLE_ANALYZER_TOKEN &&
-      new BundleAnalyzerPlugin({ token: process.env.BUNDLE_ANALYZER_TOKEN }),
-  ].filter(p => !!p),
+  plugins: [plugins.duplicatesReport].filter(p => !!p),
```

Comment:
So we retain the dupe report in CI, but we lose the reporting around bundle size over time 👍

Reply:
Yep. Those are two separate services.